nunca-ouvi-falar-de-rust/index.en.org

```diff
@@ -840,41 +847,47 @@ $ cargo run
    - Language extensions (Gems, npm, python extensions, FFI)
 
      #+REVEAL_HTML:  <small>*IMHO</small>
 
    #+reveal: split
-   [[http://www.arewelearningyet.com/][Machine Learning]], [[https://usehelix.com/][Ruby]], [[https://github.com/mitsuhiko/snaek][Python]], [[https://github.com/mgattozzi/curryrs][Haskell]], [[https://www.neon-bindings.com/][Node.js]], [[https://hackernoon.com/compiling-rust-to-webassembly-guide-411066a69fde][WebAssembly]], [[https://blogs.oracle.com/developers/building-a-container-runtime-in-rust][Container]], [[https://tokio.rs/][Network Platform]], [[https://github.com/rust-embedded][Embedded]], Operating Systems^{[[https://intermezzos.github.io/book/][1]]}^{[[https://www.redox-os.org/][2]]}^{[[https://os.phil-opp.com/][3]]}, [[http://arewegameyet.com][Games]], [[https://github.com/japaric/rust-cross][cross compilation]], [[https://github.com/japaric/trust][CI]]...
+   [[http://www.arewelearningyet.com/][Machine Learning]], [[https://usehelix.com/][Ruby]], [[https://github.com/mitsuhiko/snaek][Python]], [[https://github.com/mgattozzi/curryrs][Haskell]], [[https://www.neon-bindings.com/][Node.js]], [[https://hackernoon.com/compiling-rust-to-webassembly-guide-411066a69fde][WebAssembly]], [[https://blogs.oracle.com/developers/building-a-container-runtime-in-rust][Container]], [[https://tokio.rs/][Network Platform]], [[https://github.com/rust-embedded][Embedded]], Operational Systems^{[[https://intermezzos.github.io/book/][1]]}^{[[https://www.redox-os.org/][2]]}^{[[https://os.phil-opp.com/][3]]}, [[http://arewegameyet.com][Games]], [[https://github.com/japaric/rust-cross][cross-compilation]], [[https://github.com/japaric/trust][CI]]...
 
-* I hope that you've liked Rust
+* I hope you liked Rust
 
-** And if you want to keep learning about it
+** And if you want to learn more
 
-   - There's a [[https://doc.rust-lang.org/book/][free book]] on Rust's website
-   - [[https://riot.im/app/#/room/#mozilla_#rust-beginners:matrix.org][#rust-beginners]] is a great channel for quickly answering questions
-   - There's [[https://riot.im/app/#/room/#mozilla_#rust-br:matrix.org][#rust-br]] for Portuguese speakers
-   - The [[https://telegram.me/rustlangbr][@rustlangbr]] group on Telegram
+   - There is a [[https://doc.rust-lang.org/book/][free book]] on the website [EN]
+   - [[https://riot.im/app/#/room/#mozilla_#rust-beginners:matrix.org][#rust-beginners]] is a great channel to help with initial doubts
+   - There is [[https://riot.im/app/#/room/#mozilla_#rust-br:matrix.org][#rust-br]] for the Portuguese speakers
+   - [[https://telegram.me/rustlangbr][@rustlangbr]] on Telegram
    - [[https://www.meetup.com/topics/rust/][Meetups]]
-   - [[http://exercism.io/languages/rust/about][Exercism]]
-   - And [[https://github.com/rust-br/estudos][online study group]] in Portuguese
+   - [[http://exercism.io/languages/rust/about][Coding exercises]]
+   - And an [[https://github.com/rust-br/estudos][online studygroup]] in Portuguese
```

Comment:
"studygroup" isn't a word in English -- is there a reason you removed the space here?

Reply:
I'll fix it back hehe